libraries/chain/include/eosio/chain/transaction.hpp

```diff
@@ -166,18 +166,18 @@ namespace eosio { namespace chain {
 
       packed_transaction() = default;
 
-      explicit packed_transaction(const transaction& t, compression_type _compression = none)
+      packed_transaction(const transaction& t, compression_type _compression = none)
```

Comment:
we shouldn't remove the explicit specifier.  It was added explicitly 😏 .  We should fix the invocations where this is the correct conversion by making those conversions explicit and, where this is not the correct conversion we should remove the need for the conversion.

Reply:
Originally, I had no `explicit` modifier expecting that the second parameter would keep this from being used as an implicit cast.  However, at least on llvm4 it would use the defaulted second parameter and deploy the implicit cast.  Learned something new that day.